main.sh

```diff
@@ -17,6 +17,11 @@ title="${INPUT_TITLE:?}"
 changelog="${INPUT_CHANGELOG:-}"
 draft="${INPUT_DRAFT:-}"
 branch="${INPUT_BRANCH:-}"
+prefix="${INPUT_PREFIX:-}"
+
+# `prefix` is used as part of a regular expression, but it's a literal string,
+# so escape any regex special characters first
+prefix="$(printf '%s' "$prefix" | sed 's/[]\\.$*{}|+?()[^-]/\\&/g')"
```

Comment:
I would like to reject these special characters, to avoid future regular expression support causing a breaking change. (If we don't support regular expressions in this PR)

It seems some of them are allowed in git tag, but I think it is very rare that they are actually used (I've never seen them).